src/components/EmojiPicker/EmojiPickerMenu/index.js

```diff
@@ -384,7 +371,6 @@ function EmojiPickerMenu(props) {
         }
 
         const calculatedOffset = Math.floor(headerIndex / CONST.EMOJI_NUM_PER_ROW) * CONST.EMOJI_PICKER_HEADER_HEIGHT;
-        emojiListRef.current.flashScrollIndicators();
```

Comment:
Why was this line removed?

Reply:
`flashScrollIndicators` method is not supported in FlashList